Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -854,14 +1013,18 @@ buildMetaData req personId = do
         MD.updatedAt = now
       }
 
-makeDriverInformationRes :: DriverEntityRes -> DM.Merchant -> Maybe (Id DR.DriverReferral) -> DriverStats -> DriverInformationRes
-makeDriverInformationRes DriverEntityRes {..} org referralCode driverStats =
-  DriverInformationRes
-    { organization = DM.makeMerchantAPIEntity org,
-      referralCode = referralCode <&> (.getId),
-      numberOfRides = driverStats.totalRides,
-      ..
-    }
+makeDriverInformationRes :: (MonadFlow m, CacheFlow m r) => DriverEntityRes -> DM.Merchant -> Maybe (Id DR.DriverReferral) -> DriverStats -> DDGR.CachedGoHomeRequest -> m DriverInformationRes
+makeDriverInformationRes DriverEntityRes {..} org referralCode driverStats dghInfo = do
+  CQGHC.findByMerchantId org.id >>= \cfg ->
+    return $
+      DriverInformationRes
+        { organization = DM.makeMerchantAPIEntity org,
+          referralCode = referralCode <&> (.getId),
+          numberOfRides = driverStats.totalRides,
+          driverGoHomeInfo = dghInfo,
+          isGoHomeEnabled = cfg.enableGoHome,
```

Comment:
isGoHomeEnabled states whether this feature is enabled or not. So this will tell frontend that this feature is enabled consume the goHome data we are sending or it will tell its disabled dont consume any go home data sent.